adagrams/game.py

```diff
@@ -1,11 +1,105 @@
+import random
+
+
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    drawn_letters = [] 
+    for letter in range(10):
+        my_current_key = random.choice(list(LETTER_POOL))
+        my_current_value = LETTER_POOL.get(my_current_key)
```

Comment:
We typically only use the dictionary's .get() method when we can't be sure the key is in the dictionary. On line 36, we are choosing a key that is from the dictionary so we know it's there. As a result, it would be slightly more appropriate to just use LETTER_POOL[my_current_key] here.